cinema/views.py

```diff
@@ -58,6 +60,18 @@ class CinemaHallViewSet(
     permission_classes = (IsAdminOrIfAuthenticatedReadOnly,)
 
 
+class ItemViewSet(viewsets.ModelViewSet):
+    @action(detail=True, methods=["POST"], url_path="upload-image")
+    def upload_image(self, request, pk=None):
+        item = self.get_object()
+        serializer = self.get_serializer(item, data=request.data)
+
+        if serializer.is_valid():
+            serializer.save()
+            return Response(serializer.data, status=status.HTTP_200_OK)
+        return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+
```

Comment:
The `ItemViewSet` class appears to be unrelated to the rest of the application and lacks context. Consider reviewing its relevance and removing it if it is not needed.